Homework 1/DuodecimalFinder.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Text;
+
+namespace Homework_1
+{
+    internal class DuodecimalFinder
+    {
+        public DuodecimalFinder()
+        {
+
+        }
+
+        public string GetNumbersWhichContainSubstring(int num01,
+            int num02)
+        {
+            return string.Join(", ", FindNumbersWhichContainSubstring(num01,
+                num02));
+        }
+
+        private string FindNumbersWhichContainSubstring(int num01,
+            int num02)
+        {
+            var sb = new StringBuilder();
+            var max = Math.Max(num01, num02);
+            var min = Math.Min(num01, num02);
+            for (int i = min; i <= max; i++)
+            {
+                if (ContainsTwoAs(i))
+                {
+                    sb.Append($"{i}, ");
+                }
+            }
+            return sb.Remove(sb.Length - 2, 2).ToString();
+        }
+
+        private bool ContainsTwoAs(int number)
+        {
+            var remainderCount = 0;
+            while (number > 0)
+            {
+                var remainder = number % 12;
+                number = number / 12;
+                if (remainder == 10)
+                {
+                    remainderCount++;
+                }
```

Comment:
Bug! I have to find exactly 2 A, you can't return before the whole number will be processed. Sample- f we have value like "AA11AA" in duodecimal, you will return true but this is not suitable number